lib/core/src/Cardano/Wallet/Primitive/AddressDiscovery.hs

```diff
@@ -11,8 +11,8 @@
 -- This module contains types for address discovery. The two address discovery
 -- schemes implemented are:
 --
---  * 'Cardano.Wallet.Primitive.AddressDiscovery.Sequential'
---  * 'Cardano.Wallet.Primitive.AddressDiscovery.Random'
+--  * "Cardano.Wallet.Primitive.AddressDiscovery.Sequential"
+--  * "Cardano.Wallet.Primitive.AddressDiscovery.Random"
```

Comment:
I thought that using a single quote would cause a link to be generated?

Or does this not work for modules?

Reply:
It's what the haddock manual says. With single quotes, the identifiers need to be in scope.